WebDriverAgentLib/Categories/XCUIElement+FBUID.m

```diff
@@ -33,6 +34,19 @@ - (NSString *)fb_uid
 
 @implementation FBXCElementSnapshotWrapper (FBUID)
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wobjc-load-method"
++ (void)load
+{
+  Class XCElementSnapshotCls = objc_lookUpClass("XCElementSnapshot");
+  if (XCElementSnapshotCls != nil)
```

Comment:
I would rather let it fail in such case because other features would anyway stop working if this property is not assigned successfully

Reply:
Replaced with `NSAssert`